packages/@ember/-internals/views/lib/system/event_dispatcher.js

```diff
@@ -382,6 +382,20 @@ export default EmberObject.extend({
               if (actionHandler(target, event) === false) {
                 break;
               }
+            } else if (event.type === 'click' && target.tagName.toLowerCase() === 'a') {
+              event.preventDefault();
+              event.stopPropagation();
+
+              let isSameOrigin = window.location.origin === target.origin;
+
+              if (isSameOrigin) {
```

Comment:
this may not work if 2+ ember apps running in same page (aka "microfrontends"), should we check is route registered in app or not before transition?

Reply:
how does this work with link-to?

Reply:
maybe I should do a `recognize` before `transitionTo`

Reply:
I think link-to don't work with it, but `a` will invoke default browser behavior for same origin,  `/app/1/` ->  `/app/2`

Reply:
updated. 

_however_. 

I'm not sure how to test this additional behavior 🤔 maybe I can use sinon, or some other stubbing on the window? idk. I'll need to look around at the other link tests